app/models.py

```diff
@@ -352,8 +352,8 @@ class Organisation(db.Model):
 
     services = db.relationship(
         'Service',
-        secondary='organisation_to_service',
-        uselist=True)
+        uselist=True,
+        backref=db.backref('services'))
```

Comment:
to be consistent with other models, i don't think we should have this relationship here, instead just define it on service model only.